packages/react/src/components/Authenticator/SignUp/SignUp.tsx

```diff
@@ -100,8 +103,9 @@ SignUp.AliasControl = ({
   const {
     components: { Input, Label, Text, ErrorText },
   } = useAmplify('Authenticator.SignUp.Password');
-  const [{ context }] = useAuth();
-  const error = context.validationError[name];
+  const [state] = useAuth();
+  const { validationError } = getActorContext(state);
+  const error = validationError[name];
```

Comment:
Why is `useAuth()` sufficient for providing `state` here, but we need to use the helper method `getActorState` in other components?

Reply:
Good question -- `getActorState` is mostly used for matching states, e.g. `actorState.matches('signUp.confirmSignUp')`. Otherwise, `getActorContext` is enough to get the contexts.